tests/e2e/unified_test.go

```diff
@@ -123,7 +125,9 @@ func testFixturesInSeries(ctx context.Context, t *testing.T, testPause bool, can
 
 	subtestTimeout := time.Hour
 	if targetGCP := os.Getenv("E2E_GCP_TARGET"); targetGCP == "mock" {
-		subtestTimeout = time.Minute
+		// We allow a two minutes for the test itself (particularly deep chains with retries),
+		// and kube-apiserver has a 1 minute timeout configured
+		subtestTimeout = 3 * time.Minute
```

Comment:
I was looking at the timeout settings too at some point. FYI, it gets rebound here:

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/aa688ee381e87b732426429528bd754eed467e77/tests/e2e/unified_test.go#L124-L127

Reply:
I _think_ that's this line?

Reply:
🤦🏼 oh I thought I was reading the code in the harness